framework/src/modules/chain/transactions/transactions_handlers.js

```diff
@@ -128,6 +128,8 @@ const applyGenesisTransactions = storage => async (
 	}
 
 	const transactionsResponses = transactions.map(transaction => {
+		// Fee is handled by Elements now so we set it to zero here. LIP-0012
+		transaction.fee = new BigNum(0);
```

Comment:
Why is this done here but not in `applyTransactions` method?

Reply:
applyGenesisTransactions() behaves a little different than regular transactions so that's why this was only made here.